uds/translator/translator.py

```
@@ -92,20 +92,21 @@ def encode(self,
         """
-        if message.payload[0] == ResponseSID.NegativeResponse:
-            sid = message.payload[1]
-            return self.services_mapping[sid].decode_negative_response(message.payload)
-        sid = message.payload[0]
+        validate_raw_bytes(payload, allow_empty=False)
+        if payload[0] == ResponseSID.NegativeResponse:
+            sid = payload[1]
+            return self.services_mapping[sid].decode_negative_response(payload)
+        sid = payload[0]
         if sid not in self.services_mapping:
             raise ValueError("Database has no decoding defined for SID/RSID value of the provided message.")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)